src/Input/ToggleSwitch/ToggleSwitch.test.tsx

```diff
@@ -49,6 +49,27 @@ describe('<ToggleSwitch /> prop iconOptions', () => {
   });
 });
 
+describe('<ToggleSwitch /> prop active', () => {
+  const activeOptions = [false, true];
+  const matchSnapshot = (active: boolean) => {
+    it(`should match snapshot when active ${active} is passed`, () => {
+      const { asFragment } = render(
+        <ToggleSwitch
+          active={active}
+          iconOptions={{
+            active: EyeIcon,
+            inactive: EyeSlashedIcon,
+          }}
+        />
+      );
+      // validate defaultActive from snapshot
+      expect(asFragment()).toMatchSnapshot();
```

Comment:
Haha I just wanted to bring up this morning that maybe we should stop relying too much on snapshot tests? They are quite brittle and I find myself often just running `yarn test -u` whenever they fail so i'm not sure how much protection they can provide as well. https://www.sitepen.com/blog/snapshot-testing-benefits-and-drawbacks

In this case the test case you added below should provide enough coverage

Reply:
I agreed snapshot is unreadable, but it can help us catch the bug you pointed above https://github.com/glints-dev/glints-aries/pull/568#discussion_r553725361. So I will leave it for now. 🙏